app/scripts/controllers/eventDetails.js

```diff
@@ -79,11 +79,18 @@ angular
     $scope.conference = angular.copy(conference);
     $scope.currencies = currencies;
 
-    // If by chance the conference currency code doesn't exist
+    // If by chance the conference currency or currency code doesn't exist
     // Set the default to USD.
-    if (!$scope.conference.currency.currencyCode) {
-      $scope.conference.currency.currencyCode = 'USD';
-      $scope.saveEvent();
+    if (
+      !$scope.conference.currency ||
+      !$scope.conference.currency.currencyCode
```

Comment:
I'm not sure if this is actually necessary, because if currency is null or undefined then this would be too. But maybe there is a chance currency is defined but the currencyCode didn't get set? I'm not sure, let me know what you think.